tools/test/psa/test_find_secure_image.py

```diff
@@ -1,57 +0,0 @@
-# Copyright (c) 2019 ARM Limited
```

Comment:
If you are keeping the `find_secure_image()` function, shouldn't we also keep the test?

Reply:
Yes, missed that. I'll bring back the test.